Add playground for development (#10391)
sveltejs/kit: CONTRIBUTING.md

pnpm install
```

## Testing Changes

### Playground

> You can use the playground at [`playgrounds/basic`](./playgrounds/basic/) to experiment with your changes to Kit locally.

Reviewer: ```suggestion
You can use the playground at [`playgrounds/basic`](./playgrounds/basic/) to experiment with your changes to SvelteKit locally.
```

### Linking

> If you don't want to use the playground you can use [pnpm `overrides`](https://pnpm.io/package_json#pnpmoverrides) to link it to your existing project:

Reviewer: ```suggestion
If you want to test against an existing project, you can use [pnpm `overrides`](https://pnpm.io/package_json#pnpmoverrides) in that project:
```
